### DISCUSSION TITLE
installation guide / process

### DISCUSSION THREAD

**Discussion by Suplexxer** [nod-ai/SHARK#874] [Q&A]
Hey all
first off, thank you for the .exe file for installation. Easiest installation of any text2img AI by far.
a quick note on my setup, if its ever relevant.
amd rysen 9 5950, amd RX6950XT, 32bg ram. 
got python 3.10, GIT, Conda 
(getting roughly 4,8 iterations/sec)

to the question. 
what are the effects of just running the .exe installation file and never downloading the code base, or setting up /downloading dependencies like IREE, PIP, Torch etc? im not familiar at all in this dev lite setup. 
So the question comes more down to, am i missing features by not installing the binary first, like i see that contributors are working on img2img feature will that be available in the upcoming .EXE installation setup or some other way?
Please let me know if you dont understand my question.

i am getting errors running text2img

`WARNING: [Loader Message] Code 0 : windows_read_data_files_in_registry: Registry lookup failed to get layer manifest files.`
anything i should fix?

Another point.
Currently the install guide feels like (it probably isent)
1. download <these things>
2. install the stuff with the arguments you like
3. ???
4. profit

if someone would explain the steps a bit more that would make the things easier for beginners.

Altho if the .EXE is all you need then this is a moot point and you should ju put that text up front i the readme. "dont bother with installing anything, just run the .exe file and you can do whatever and all upcoming features will be included."

Any help is appreciated
/Sup

**Comment by Morello-Reanadore** [comment]
I keep getting errors, why is the installation process so difficult?
